package TryCatch;

public class NumeroNegativo2Exception  extends RuntimeException{

        public NumeroNegativo2Exception(){
            super("Raiz negativa. Entre com valores n√£o negativos!");
        }


}
